templates/modern-solutions/dop/Makefile

```diff
@@ -0,0 +1,16 @@
+all: dop payload
+
+.PHONY: clean
+
+CC = clang
+CFLAGS = -static
+#CFLAGS = -fsanitize=safe-stack
+
+dop: dop.c
+	$(CC) $(CFLAGS) -o $@ $<
+
+payload: payload.c
+	$(CC) $(CFLAGS) -o $@ $<
+
+clean:
+	rm -f dop payload *.o
```

Comment:
`-rm ...`

Reply:
-rm? Did you mean the -r flag? -m doesn't exist.